docs/core/docker/publish-as-container.md

```diff
@@ -13,10 +14,21 @@ Containers have many features and benefits, such as being an immutable infrastru
 
 Install the following prerequisites:
 
+:::zone pivot="dotnet-8-0"
+
+- [.NET 8+ SDK](https://dotnet.microsoft.com/download/dotnet/8.0)\
+If you have .NET installed, use the `dotnet --info` command to determine which SDK you're using.
+- [Docker Community Edition](https://www.docker.com/products/docker-desktop)
+
+:::zone-end
+:::zone pivot="dotnet-7-0"
```

Comment:
These pivots are really interesting - in the in-repo docs we have a lot of "starting in...." text that can just go away through use of  .NET 8 pivots in this doc!